### PR TITLE
Fix typo in debian/changelog

### PR DESCRIPTION
**Pull request by eLvErDe**
Two spaces are required after email addr in debian/changelog. "debuild" fails with syntax error before fixing it

### CODE REVIEW

**Comment by eLvErDe** [comment]
Various Debian packaging improvment

File: debian/control [hunk 1]
```diff
@@ -11,11 +11,10 @@ Homepage: http://www.packetfence.org/
 Package: packetfence
 Architecture: all
 # TODO: We can probably move these in Depends since 3.5.0 (managed RADIUS feature)
-Pre-Depends:  ca-certificates, freeradius (>= 2.2.5), freeradius-ldap, freeradius-postgresql,
```
Comment (fdurand) [inline debian/control:14]: We need freeradius as predepend for the live-build

**Comment by eLvErDe** [comment]
Hello,

What do you mean by "pre-build". I,ve been able to build the package successfully with these patches.
Pre-depends is a very specific need: it means this package need the other one BEFORE running its own preinst script. It's supposed to be very rare :-)

Tell me more and I will try to improve if you're interested in.

Regards,

**Comment by fdurand** [comment]
Yes, you can build the packet and install it correctly in debian/ubuntu , but it´s in the case we build the live-cd and live-usb (http://live.debian.net/devel/live-build/) we must have freeradius as a pre-dep

File: debian/control [hunk 2]
```diff
@@ -11,11 +11,10 @@ Homepage: http://www.packetfence.org/
...
- freeradius-mysql, freeradius-krb5, dhcp3-server
+Pre-Depends:  ca-certificates
 Breaks: libdata-alias-perl
 Depends: ${misc:Depends}, vlan, make,
- openssl, openssl-blacklist, openssl-blacklist-extra, snort | suricata,
```
Comment (fdurand) [inline debian/control:18]: Good idea

File: debian/rules [hunk 3]
```diff
@@ -158,13 +158,14 @@ binary-arch: build install
 		-X$(NAME)sbin/pfarp_remote
 	chmod 2775 $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/var/run
 	chmod 2775 $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/var/dhcpd
-	chown dhcpd:dhcpd $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/var/dhcpd/dhcpd.leases
-	# Set setuid root on bin/pfcmd
-	chown pf:pf -R $(CURDIR)/debian/packetfence-pfcmd-suid$(PREFIX)/$(NAME)/
+	# Nop, dhcpd won't exists on the build machine
+	# Maybe it should be done by the postinst script ?
```
Comment (fdurand) [inline debian/rules:162]: posinst i prefer

**Comment by dwlfrth** [comment]
Where Am I ?

**Comment by julsemaan** [comment]
What's the status on this ?

**Comment by julsemaan** [comment]
@fdurand Has this been merged in a separate pull request ?

**Comment by fdurand** [comment]
This has been merged in another pull request so i close it

**Pull request closed without merging**